core/rfb.js

```diff
@@ -790,8 +797,15 @@ export default class RFB extends EventTargetMixin {
 
         if (this._viewOnly) { return; } // View only, skip mouse events
 
-        if (this._rfb_connection_state !== 'connected') { return; }
-        RFB.messages.pointerEvent(this._sock, this._display.absX(x), this._display.absY(y), this._mouse_buttonMask);
+        // Limit mouse move events to one every MOUSE_MOVE_DELAY mS
+        clearTimeout(this.timerID);     // Remove old one
```

Comment:
`timerID` is a bit generic. Please include something that identifies what timer it is.

Reply:
fixed